doc/nrf/gs_modifying.rst

```diff
@@ -84,55 +93,116 @@ The following CMake commands can be managed by SES, if they target the ``app`` l
 
 The :file:`CMakeLists.txt` files for the sample applications in the :file:`sdk-nrf` repository already have the required tags.
 Therefore, if you always use SES to maintain them, you do not need to worry about tagging.
-Typically, the :file:`CMakeLists.txt` files include at least the :file:`main.c` file as source::
+Typically, the :file:`CMakeLists.txt` files include at least the :file:`main.c` file as source:
+
+.. code-block:: c
 
    # NORDIC SDK APP START
    target_sources(app PRIVATE src/main.c)
    # NORDIC SDK APP END
 
+Advanced compiler settings
+==========================
+
+The application has full control over the build process.
+
+Using Zephyr's configuration options is the standard way of controlling how the system is built.
+These options can be found under Zephyr's menuconfig :guilabel:`Build and Link Features` > :guilabel:`Compiler Options`.
+For example, to turn off optiomizations, select :kconfig:`CONFIG_NO_OPTIMIZATIONS`.
+
+Compiler options not controlled by the Zephyr build system can be manually controlled through the :kconfig:`CONFIG_COMPILER_OPT` Kconfig option.
```

Comment:
I know I proposed the sentence myself, but I think `manually` can be omitted (didn't notice that at first).
All Kconfig options is kind of manual.

```suggestion
 Compiler options not controlled by the Zephyr build system can be controlled through the :kconfig:`CONFIG_COMPILER_OPT` Kconfig option.
```

Reply:
Edited.